server/apps/archive/common.py

```diff
@@ -399,9 +412,19 @@ def item_schema(extra=None):
             'type': 'datetime',
             'nullable': True
         },
-        'marked_for_not_publication': {
-            'type': 'boolean',
-            'default': False
+        'flags': {
+            'type': 'dict',
+            'nullable': True,
```

Comment:
Since these are boolean values I think it's better to have non-nullable value. By allowing nullable we are missing the requirement "By default all items are marked for published"